app/views/directives/osc-source-secrets.html

```diff
@@ -59,9 +58,9 @@
         <div class="row advanced-secrets">
           <div class="col-lg-6">
             <label class="picker-label" ng-if="$first">Build Secret</label>
-            <ui-select ng-required="pickedSecret.mountPath !== ''" ng-model="pickedSecret.secretSource.name">
+            <ui-select ng-model="pickedSecret.secretSource.name">
```

Comment:
Don't we still want to make the secret name required if a mount path is entered?